### ISSUE TITLE
Questions aboud segmentation head

### ISSUE THREAD

**Issue by XiHuYan** [rshwndsz/hover-net#2]
```
class _SegmentationHead(nn.Module):
    def __init__(self, head):
        super(_SegmentationHead, self).__init__()
        assert head in ['np', 'hv'], "Head must be 'np' or 'hv'"
        self.head = head

        self.bn_relu = _BNRelu(num_features=64)
        self.conv1 = nn.Conv2d(64, 2, kernel_size=1,
                               padding=0, stride=1, dilation=1, bias=True)

    def forward(self, inputs):
        out = self.bn_relu(inputs)
        out = self.conv1(out)
        if self.head == 'np':
            out = F.softmax(out, dim=1)
            out = torch.max(out, dim=1)[0].unsqueeze(dim=1)

        return out
```
when head is "np", torch.max will return (values, index), if you want to get the segmention results,  
why not use `torch.max(out, dim=1)[1]` to return 'index'?

Maybe you should carefully check you code.
